commons/passive/empaticaembraceplus/empaticaembraceplus_oxygen_saturation.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.passive.empaticaembraceplus",
+  "type": "record",
+  "name": "EmpaticaEmbracePlusOxygenSaturation",
+  "doc": "Blood oxygen saturation (SpO2) during rest.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+    { "name": "spo2Percentage", "type": "int", "doc": "SpO2 level (percentage)." },
```

Comment:
Similarly "percentage" can be removed from the name here. Units can be removed from the names as they are part of the docs.